osmomath/decimal.go

```diff
@@ -305,6 +305,17 @@ func (d BigDec) MulTruncate(d2 BigDec) BigDec {
 	return BigDec{chopped}
 }
 
+// multiplication round up
+func (d BigDec) MulRoundUp(d2 BigDec) BigDec {
```

Comment:
wait how does this ever affect anything?

Multiplication has no rounding under decimal?